okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -52,6 +66,8 @@ class HttpLoggingInterceptor
     @Volatile
     var level = Level.NONE
 
+    var requestLevel: (Request) -> Level = { level }
```

Comment:
This isn’t very Java language friendly. I’ve tried to avoid types like `Function1` in our public APIs

Reply:
(We have overloads in OkHttpClient.Builder, but there’s also a Java way to get at those APIs.)